src/screens/ENSIntroSheet.tsx

```diff
@@ -186,7 +187,9 @@ export default function ENSIntroSheet() {
                 </Stack>
                 <Stack space={{ custom: isSmallPhone ? 30 : 40 }}>
                   <Bleed left="10px">
-                    <IntroMarquee isSmallPhone={isSmallPhone} />
+                    {IS_TESTING !== 'true' && (
```

Comment:
It's not a huge deal, don't need to update this PR, but we can start using `IS_TEST` from `@/env` in place of this everywhere.

Reply:
We can definitely do it already here.